packager/react-packager/src/Server/index.js

```diff
@@ -402,7 +405,28 @@ class Server {
     const assetEvent = Activity.startEvent(`processing asset request ${assetPath[1]}`);
     this._assetServer.get(assetPath[1], urlObj.query.platform)
       .then(
-        data => res.end(data),
+        data => {
```

Comment:
I'd extract this function to something like `sendWithRange` module for easier understanding.